how-to-guides/mainnet.md

```diff
@@ -44,18 +44,20 @@ and service availability, visit the
 
 ### Transaction size limit
 
-As specified in [CIP-28](https://cips.celestia.org/cip-028.html), there is a 2 MiB (2,097,152 bytes) limit on individual transaction size. This limit was implemented to maintain network stability and provide clear expectations for users and developers, even as block sizes may be larger.
+Currently, the maximum transaction size is 8 MiB (8,388,608 bytes) as of [CIP-38](https://cips.celestia.org/cip-038.html) going live on the Matcha v6 upgrade.
+
+Previously, in [CIP-28](https://cips.celestia.org/cip-028.html), there was a 2 MiB (2,097,152 bytes) limit on individual transaction size. This limit was implemented to maintain network stability and provide clear expectations for users and developers, even as block sizes may be larger.
 
 ### Block size limit
 
-While individual transactions are limited to 2 MiB, a block can contain multiple transactions and has a much larger capacity. The maximum block size is determined by the effective maximum square size.
+While individual transactions are limited to 8 MiB as of v6, a block can contain multiple transactions and has a much larger capacity. The maximum block size is determined by the effective maximum square size.
 
-Given that the current governance maximum square size is 128, the total block size can be slightly less than ~8 MiB, or 7,896,602 bytes to be exact.
+Given that the current governance [maximum square size is 512](https://cips.celestia.org/cip-038.html#parameters), the total block size can be slightly less than ~120.5 MiB (126,352,922 bytes), based on the share layout used for blobs.
```

Comment:
```suggestion
The current max square size on Arabica is 128 MiB, Mocha is 32 MiB, and Mainnet is 8 MiB.
```

Reply:
this seems straightforward enough, thank you again